src/playground-code-editor.ts

```diff
@@ -777,7 +799,9 @@ export class PlaygroundCodeEditor extends LitElement {
       // Note there is no API for "select the next naturally focusable element",
       // so instead we just re-focus the outer container, from which point the
       // user can tab to move focus entirely elsewhere.
```

Comment:
`_completionsOpen` implementation is awesome. May be helpful to add to this comment that when the user is escaping out of completions, we do not want to move focus away from the CodeMirror editor.